applications/nublado/templates/controller-serviceaccount.yaml

```diff
@@ -53,6 +53,22 @@ rules:
       - "get"
       - "list"
       - "watch"
+  - apiGroups: ["batch"]
+    resources: ["jobs"]
+    verbs:
+      - "create"
+      - "delete"
+      - "get"
+      - "list"
+      - "watch"
+  - apiGroups: ["networking.k8s.io"]
+    resources: ["networkpolicies", "ingresses"]
```

Comment:
`networkpolicies` is already handled below and shouldn't be listed here.

Reply:
Rather, I should delete the below stanza, since networkpolicies and ingresses want the same verbs.